nipype/interfaces/mrtrix3/utils.py

```diff
@@ -560,7 +560,7 @@ class MRConvertInputSpec(MRTrix3BaseInputSpec):
         desc="output image",
     )
     coord = traits.List(
-        traits.Float,
+        traits.Int,
```

Comment:
If previous versions allowed floats, then this change could break workflows that were written for those versions. Do you know what the history is here?